### PR TITLE
Resolve #25 -- Adjust git repos for cloning

### LINKED ISSUE #25: Adjust branches per repo
The LeaguePackets repo don't have a Indev branch, let's set branch per repo while cloning.

### PR DESCRIPTION
**Pull request by Kipero**
The LeaguePackets repo don't have a Indev branch, let's set branch per repo while cloning

Resolve #25

### CODE REVIEW

File: League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs [hunk 1]
```diff
@@ -134,19 +134,19 @@ private void startCloningRepositories()
...
                 if (!Directory.Exists(cloningPath))
                 {
                     Directory.CreateDirectory(cloningPath);
+                    options.BranchName = "Indev";  // Branch for GameServer
                     Repository.Clone("https://github.com/LeagueSandbox/GameServer", cloningPath, options);
-                    options.BranchName = "master";
+                    options.BranchName = "master"; // Branch for LeaguePackets
                     Repository.Clone("https://github.com/LeagueSandbox/LeaguePackets", Path.Combine(cloningPath, "LeaguePackets"), options);
-                    options.BranchName = "indev";
+                    options.BranchName = "Indev"; // Branch for LeagueSandbox-Default
```
Comment (danil179) [inline League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs]: I'm not sure why you have changed `indev->Indev`. Git branch names in general I think are case sensitive, but seems that this is not the case on Windows. I suggest to change `Indev->indev`.

**Review by danil179 — COMMENTED** [review]
Comment about branch names.

File: League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs (hunk 1 above)
Comment (Kipero) [reply League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs]: Done, could we switch whole repos to use master / **main** branch, as the most stables ones for future development?

Comment (lzardy) [reply League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs]: There is no need to switch.

**Review by danil179 — APPROVED** [review]
Approved.

**Pull request merged**